### PR TITLE
Really remote pry-remote

### PR DESCRIPTION
**Pull request by aried3r**
Sorry, my fault. 😢

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,7 +73,6 @@ mixed encodings.
 [awesome_print]:      https://github.com/michaeldv/awesome_print
 [hirb]:               https://github.com/cldwalker/hirb
 [pry-doc]:            https://github.com/pry/pry-doc
-[pry-remote]:         https://github.com/Mon-Ouie/pry-remote
```
Comment (plribeiro3000) [inline README.md:77]: I think you need to leave this in here because of other references.

Comment (aried3r) [inline README.md:77]: The README.md has no further references.

Comment (plribeiro3000) [inline README.md:77]: Oh, you are right.

**Comment by plribeiro3000** [comment]
Releases as `4.0.1`.

Thanks for your quick fix.

**Pull request merged**